doc/manual/migrating_from_eeglab.rst

```diff
@@ -0,0 +1,68 @@
+Caveats coming from EEGLAB
+==========================
+
+EEGLAB vs. MNE: Pros and Cons
+-----------------------------
+
+Major differences: MNE-Python ...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* is **Python based**. Python is a fully featured, general programming language. It is easy to learn, easy to read (and thus share, criticize and reproduce code with). It is also free - unlike MATLAB, which you need for EEGLAB, and the MATLAB toolboxes. For example, with MNE, you will never run out of/lack parallel processing toolboxes, and thus MNE more readily uses multiple cores (usually with the `n_jobs` keyword).
+* was born as a package for **MEG analysis**. While *MNE has gained strong EEG support*, it in many instances takes a more extensive approach. For example, MEG data is usually multimodal (including magnetometers, gradiometers and EEG), and thus MNE requires you to constantly choose what channel type you want to operate with. MNE also has much more integrated support for source-level/source localisation than EEGLAB.
+* is **Object oriented**. Typical user-exposed functions don't operate on raw data matrices, but on `Raw`, `Epochs`, `Evoked`, ... objects. Many object methods operate in-place. For example, when in MNE, you do EEG = pop_eegfiltnew(EEG, ...), in MNE, you do raw.filter(...). In EEGLAB, you do pop_plot_topo(EEG, ...). In MNE, you do evoked.plot_topomap(...). In EEGLAB, you do pop_saveset(EEG, ...). In MNE, you do `epochs.save(...)`. `raw.plot()` is very different from `evoked.plot()`. In-place modification can usually be turned off with the `copy=True` setting.
+* does **not feature a GUI**. MNE always requires explicit code. This can be frustrating for beginners, but in the long run, it supports good scientific practices - better reproducible, more scalable analyses. Generally speaking, there is some trade-off between the lack of GUI in MNE and the Python base; while GUIs are more accessible to newcomers, Python more easily allows experienced users to create custom and advanced analysis pipelines and techniques.
+* has a different *status and style* of the project. EEGLAB is the biggest EEG analysis package. MNE is a comparatively young project. It follows modern coding standards (including code review and unit tests). Its development is less monolithic, more distributed and democratic than that of EEGLAB. There are many published papers using EEGLAB, and very few with MNE-Python, and many users of EEGLAB and few of MNE.
```

Comment:
Please be careful, we don't want to get into politics here. I would basically drop the qualifyimg sentences that say X is better or more developped, such sentences may be taken literally by many and can be outdated very quickly. It is not good for the peace among neighbours.